lib/api/src/sys/externals/table.rs

```diff
@@ -211,3 +211,18 @@ impl<'a> Exportable<'a> for Table {
         }
     }
 }
+
+#[test]
+fn test_table_grow_3197() {
```

Comment:
Let's add a link for lazy people like me :stuck_out_tongue: 

```suggestion
/// Check the example from <https://github.com/wasmerio/wasmer/issues/3197>.
#[test]
fn test_table_grow_issue_3197() {
```